### PR TITLE
Added typing

### PR DESCRIPTION
**Pull request by tekktrik**

### CODE REVIEW

**Comment by tekktrik** [comment]
Not entirely sure how to handle this build failure, but happy to stay posted!

**Comment by tekktrik** [comment]
Added _bleio to the mock imports in docs/conf.py, just want to make sure that's the correct solution here

**Comment by dhalbert** [comment]
You can add `adafruit-blinka-bleio` to the requirements, that should work as well, and will have better checking.

**Comment by tekktrik** [comment]
Tried adding adafruit-blinka-bleio to the docs/requirements.txt in a test branch and removing the mock import and it failed again:  https://github.com/tekktrik/Adafruit_CircuitPython_AirLift/actions/runs/1453446829 

It looked like that requirement was already in the library requirements.txt too

**Comment by dhalbert** [comment]
Ah, too bad, it's because the CI machine can't run a Bluetooth utility we need:
`FileNotFoundError: [Errno 2] No such file or directory: 'bluetoothctl': 'bluetoothctl'`. OK, then just use the mock.

File: adafruit_airlift/esp32.py [hunk 1]
```diff
@@ -89,7 +97,7 @@ def __init__(
         # Used for WiFi mode.
         self._spi = spi
 
-    def reset(self, mode, debug=False):
+    def reset(self, mode: int, debug: bool = False):
```
Comment (dhalbert) [inline adafruit_airlift/esp32.py]:
```suggestion
    def reset(self, mode: int, debug: bool = False) -> None:
```

Comment (dhalbert) [reply adafruit_airlift/esp32.py]: If you don't have the `-> None`, it reverts to `-> Any`, so it's better to include it explicitly. (Same in other libraries.)

**Review by dhalbert — CHANGES_REQUESTED** [review]
Just `-> None`

File: adafruit_airlift/esp32.py (hunk 1 above)
Comment (tekktrik) [reply adafruit_airlift/esp32.py]: Ah so returns (and lack thereof) should always type hint as `None`?

Comment (dhalbert) [reply adafruit_airlift/esp32.py]: Yes. There are examples of that in the regular doc, for instance in `DigitalInOut`:
image

Comment (tekktrik) [reply adafruit_airlift/esp32.py]: Awesome, thanks for letting me know!

**Review by FoamyGuy — APPROVED** [review]
Changes look good to me. It appears all of the requested changes have been completed.

**Pull request merged**